apps/docs/src/pages/docs/status.mdx

```diff
@@ -0,0 +1,39 @@
+# Project Status
+
+**UI Kit** is actively maintained by a dedicated team committed to enabling high-quality digital product development. Stay informed about the project’s current status and ongoing development.
+
+## Design System Support
+
+Originally built on the [Next Design System](https://designsystem.hitachivantara.com/6c705d900/p/43c9a1-next-design-system), **UI Kit** has since evolved into a **design system–agnostic** library, powered by a flexible [theming system](/docs/theming) that allows teams to meet diverse branding and design requirements.
+
+Recent development has prioritized support to the Pentaho Design System, so updates for NEXT DS are temporarily paused and are not fully reflected in the current **UI Kit** version.
```

Comment:
Would it make more sense to use "deprioritized", as we eventually (indirectly) end up updating NEXT components when reviewing P+? We also mention it's a matter of priority on the start of the sentence
Also, **Pentaho+**
```suggestion
Recent development has prioritized support to the Pentaho+ Design System, so updates for NEXT DS are temporarily deprioritized and are not fully reflected in the current **UI Kit** version.
```